src/NHibernate.Test/Async/Tools/hbm2ddl/SchemaUpdate/MigrationFixture.cs

```diff
@@ -25,18 +27,36 @@ namespace NHibernate.Test.Tools.hbm2ddl.SchemaUpdate
 	[TestFixture]
 	public class MigrationFixtureAsync
 	{
-		private async Task MigrateSchemaAsync(string resource1, string resource2, CancellationToken cancellationToken = default(CancellationToken))
+		private Configuration _configurationToDrop;
+		private FirebirdClientDriver _fireBirdDriver;
```

Comment:
I am curious about approaches that are used in nhibernate to testing. Does it mean that these tests are executed only against Firebird?